internal/dataavailability/http_custom_error.go

```diff
@@ -1,18 +1,18 @@
 package dataavailability
 
 type HttpCustomError struct {
-	status uint
+	status int
```

Comment:
Sorry, but why change `uint`to `int`?

Reply:
Because `go-ethereum` uses `int` for HTTP status codes.